<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="2" column="0" rowspan="2" colspan="8">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="2" column="3">
       <widget class="Kafelek" name="kafelek_13" native="true"/>
      </item>
      <item row="2" column="6">
       <widget class="Kafelek" name="kafelek_16" native="true"/>
      </item>
      <item row="3" column="1">
       <widget class="Kafelek" name="kafelek_21" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="Kafelek" name="kafelek_30" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="Kafelek" name="kafelek_00" native="true">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="Kafelek" name="kafelek_25" native="true"/>
      </item>
      <item row="3" column="4">
       <widget class="Kafelek" name="kafelek_24" native="true"/>
      </item>
      <item row="2" column="4">
       <widget class="Kafelek" name="kafelek_14" native="true"/>
      </item>
      <item row="4" column="1">
       <widget class="Kafelek" name="kafelek_31" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="Kafelek" name="kafelek_01" native="true">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="Kafelek" name="kafelek_15" native="true"/>
      </item>
      <item row="5" column="2">
       <widget class="Kafelek" name="kafelek_42" native="true"/>
      </item>
      <item row="1" column="6">
       <widget class="Kafelek" name="kafelek_06" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="Kafelek" name="kafelek_20" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="Kafelek" name="kafelek_10" native="true"/>
      </item>
      <item row="5" column="4">
       <widget class="Kafelek" name="kafelek_44" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="Kafelek" name="kafelek_03" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Wtorek</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="Kafelek" name="kafelek_23" native="true"/>
      </item>
      <item row="4" column="6">
       <widget class="Kafelek" name="kafelek_36" native="true"/>
      </item>
      <item row="4" column="5">
       <widget class="Kafelek" name="kafelek_35" native="true"/>
      </item>
      <item row="5" column="3">
       <widget class="Kafelek" name="kafelek_43" native="true"/>
      </item>
      <item row="3" column="6">
       <widget class="Kafelek" name="kafelek_26" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Czwartek</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Kafelek" name="kafelek_11" native="true"/>
      </item>
      <item row="5" column="5">
       <widget class="Kafelek" name="kafelek_45" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Środa</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Niedziela</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="Kafelek" name="kafelek_41" native="true"/>
      </item>
      <item row="4" column="4">
       <widget class="Kafelek" name="kafelek_34" native="true"/>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Sobota</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="Kafelek" name="kafelek_33" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="Kafelek" name="kafelek_02" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Poniedziałek</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="Kafelek" name="kafelek_40" native="true"/>
      </item>
      <item row="1" column="4">
       <widget class="Kafelek" name="kafelek_04" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="Kafelek" name="kafelek_12" native="true"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Piątek</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="Kafelek" name="kafelek_05" native="true"/>
      </item>
      <item row="3" column="2">
       <widget class="Kafelek" name="kafelek_22" native="true"/>
      </item>
      <item row="4" column="2">
       <widget class="Kafelek" name="kafelek_32" native="true"/>
      </item>
      <item row="5" column="6">
       <widget class="Kafelek" name="kafelek_46" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actional_ma_kota"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_strzalka_w_gore"/>
   <addaction name="action_strzalka_w_dol"/>
   <addaction name="action_miesiac"/>
   <addaction name="action_rok"/>
  </widget>
  <action name="action_strzalka_w_gore">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/strzalkawgore.png</normaloff>:/img/strzalkawgore.png</iconset>
   </property>
   <property name="text">
    <string>strzalka_w_gore</string>
   </property>
   <property name="toolTip">
    <string>strzalka w gore</string>
   </property>
  </action>
  <action name="action_strzalka_w_dol">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/pobrane.png</normaloff>:/img/pobrane.png</iconset>
   </property>
   <property name="text">
    <string>strzalka_w_dol</string>
   </property>
   <property name="toolTip">
    <string>strzałka w dół</string>
   </property>
  </action>
  <action name="action_miesiac">
   <property name="text">
    <string>miesiac</string>
   </property>
   <property name="toolTip">
    <string>miesiac</string>
   </property>
  </action>
  <action name="action_rok">
   <property name="text">
    <string>rok</string>
   </property>
   <property name="toolTip">
    <string>rok</string>
   </property>
  </action>
  <action name="actional_ma_kota">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kafelek</class>
   <extends>QWidget</extends>
   <header>kafelek.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
